docs/docs/deploy/clever-cloud.md

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_position: 4
+title: 💎 Clever Cloud
+---
+
+# Clever Cloud 💎
+
+[Clever Cloud](https://www.clever-cloud.com/) is a PaaS provider that enables you to start apps quickly using `git` while they manage the infrastructure, scalability, and run lifetime.
+
+## Prerequisites
+
+Before you get started, if you don't already have these, you'll need to create:
+
+- A [Clever Cloud Account](https://api.clever-cloud.com/v2/sessions/signup)
+- Add billing information
+
+:::caution
+As with any PaaS service you should check the pricing before continuing. Clever Cloud has a transparent pricing policy depending on the runtime type and size, see the [Clever Cloud Pricing Page](https://www.clever-cloud.com/pricing/).
+:::
+
+You might be able to do everything in the console, but for this tutorial, we will use the [Clever Cloud CLI](https://www.clever-cloud.com/doc/reference/clever-tools/getting_started/), so you'll have to install it on your computer.
+
+Then log in running:
+
+```bash
+clever  login
+```
+
+## Deploying
+
+### 1. Create the Clever Cloud application
+
+_Clever Cloud_ has [many available runtime](https://www.clever-cloud.com/product/) that are optimized for the type of application you want to deploy. Unfortunately, **dart** is **not** _yet_ one of them.
```

Comment:
Is there a source of information (website, Github issue, etc) where users can refer to to see the progress of Clever Cloud supporting Dart?